frontend/app/src/components/history/events/HistoryEventsView.vue

```diff
@@ -447,7 +447,7 @@ watch(
 
 const premium = usePremium();
 const { isLoading: isSectionLoading } = useStatusStore();
-const sectionLoading = isSectionLoading(Section.TX);
+const sectionLoading = isSectionLoading(Section.HISTORY_EVENT);
 const eventTaskLoading = isTaskRunning(TaskType.TX_EVENTS);
```

Comment:
Since you are renaming the section property maybe it would also make sense to also update this here?